app/src/main/java/com/andryoga/safebox/data/db/secureDao/UserDetailsDaoSecure.kt

```diff
@@ -21,7 +21,7 @@ class UserDetailsDaoSecure @Inject constructor(
         return userDetailsDao.getUserDetails()
     }
 
-    override suspend fun getHint(): UserDetailsEntity {
+    override suspend fun getHint(): String {
         return userDetailsDao.getHint()
```

Comment:
where are you decrypting?

Reply:
Oh I didnt add decryption I will just do it